utils/writeable/src/utf.rs

```diff
@@ -0,0 +1,89 @@
+// This file is part of ICU4X. For terms of use, please see the file
```

Comment:
Nit: Add this Writeable to both the PR title and the CHANGELOG. I had overlooked this PR because I assumed it was a Diplomat-specific change.